packages/prisma/src/prisma/schema/js.prisma

```diff
@@ -1,15 +1,18 @@
 generator client {
-  provider        = "prisma-client-js"
-  previewFeatures = ["postgresqlExtensions", "views", "queryCompiler", "driverAdapters"]
-  output          = "../client"
-  binaryTargets   = ["native", "debian-openssl-1.1.x", "linux-musl-openssl-3.0.x", "debian-openssl-3.0.x"]
+  provider               = "prisma-client"
+  previewFeatures        = ["postgresqlExtensions", "views", "queryCompiler", "driverAdapters"]
+  output                 = "../client"
+  runtime                = "nodejs"
+  moduleFormat           = "esm"
```

Comment:
prisma client will now be built to .ts files as a clean ES module and then will be built into our own "client" with rollup